.eslintrc.json

```diff
@@ -28,7 +28,8 @@
       { "prefixWithI": "always" }
     ],
     "no-tabs": "error",
-    "react/prop-types": "off"
+    "react/prop-types": "off",
+    "@typescript-eslint/no-unused-vars": "error"
```

Comment:
I wonder if we want to make some other checks an error?

Reply:
It's likely, this is just the only one that comes to mind right now.